information-schema/information-schema-tiflash-replica.md

```diff
@@ -28,3 +28,13 @@ DESC tiflash_replica;
 +-----------------+-------------+------+------+---------+-------+
 7 rows in set (0.01 sec)
 ```
+
+Fields in the `TIFLASH_REPLICA` table are described as follows:
+
+- `TABLE_SCHEMA`: the name of the database to which the table belongs.
+- `TABLE_NAME`: the name of the table.
+- `TABLE_ID`: the internal ID of the table, which is unique within a TiDB cluster.
+- `REPLICA_COUNT`: the number of TiFlash replicas.
+- `LOCATION_LABELS`: the LocationLabelList that is set when a TiFlash replica is created.
+- `AVAILABLE`: indicates whether the TiFlash replica of the table is available. When the value is `1` (available), TiDB optimizer can intelligently choose to push down queries to TiKV or TiFlash based on query cost. When the value is `0` (unavailable), TiDB will not push down queries to TiFlash. The `AVAILABLE` does not change after it becomes `1` (available).
```

Comment:
```suggestion
- `AVAILABLE`: indicates whether the TiFlash replica of the table is available. When the value is `1` (available), TiDB optimizer can intelligently choose to push down queries to TiKV or TiFlash based on query cost. When the value is `0` (unavailable), TiDB will not push down queries to TiFlash. Once the value of this field becomes `1` (available), it will not change anymore.
```